palace/deps/patch/mfem/patch_direct.diff

```diff
@@ -183,37 +183,6 @@ index f8af98a72..0d4503723 100644
  #cmakedefine MFEM_USE_BENCHMARK
  
  // Enable Enzyme for AD
-diff --git a/config/cmake/modules/FindButterflyPACK.cmake b/config/cmake/modules/FindButterflyPACK.cmake
```

Comment:
You probably did not mean to remove this file from the patch along with `FindZFP.cmake`.